### PR TITLE
Added goal/progress vars to the dosomething_helpers config.

### PR DESCRIPTION
**Pull request by angaither**
The functionality for this  isn't finished yet, but it might get added to prod as is.  
it just adds new vars to `dosomething_helpers` in prep for #4281 & #4282

### CODE REVIEW

File: lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc [hunk 1]
```diff
@@ -142,6 +142,30 @@ function dosomething_helpers_variable_form($form, &$form_state, $node) {
...
+    $form['goals'] = array(
+      '#type' => 'fieldset',
+      '#title' => t('Goals Settings'),
+      '#collapsible' => TRUE,
+      '#collapsed' => TRUE,
+    );
+    $form['goals']['goal'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Goal'),
+      '#required' => FALSE,
```
Comment (sergiitk) [inline lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc]: I think it's not required by default. Could you please check it and remove if yes?

File: lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc [hunk 2]
```diff
@@ -142,6 +142,30 @@ function dosomething_helpers_variable_form($form, &$form_state, $node) {
...
+      '#description' => 'The goal of total number of reportbacks',
+      '#default_value' => $vars['goal'],
```
Comment (sergiitk) [inline lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc:157]: `$vars['goal']` may not be set, which could trigger undefined index notice. Pls preset its default or ensure it is set by this point.

File: lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc [hunk 3]
```diff
@@ -142,6 +142,30 @@ function dosomething_helpers_variable_form($form, &$form_state, $node) {
...
+    );
+    $form['goals']['progress'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Progress'),
+      '#description' => t('The total number of approved reportback quantity.'),
+      '#required' => FALSE,
+      '#default_value' => $vars['progress'],
```
Comment (sergii-tkachenko) [inline lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc]: Same here.

File: lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc (hunk 2 above)
Comment (sergiitk) [inline lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc:157]: Sorry, I see now. You extended `dosomething_helpers_get_variable_names()`. Pls ignore.

**Comment by angaither** [comment]
@sergii-tkachenko updated

File: lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc [hunk 4]
```diff
@@ -142,6 +142,28 @@ function dosomething_helpers_variable_form($form, &$form_state, $node) {
...
+    $form['goals'] = array(
+      '#type' => 'fieldset',
+      '#title' => t('Goals Settings'),
+      '#collapsible' => TRUE,
+      '#collapsed' => TRUE,
+    );
+    $form['goals']['goal'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Goal'),
+      '#description' => 'The goal of total number of reportbacks',
```
Comment (sergiitk) [inline lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc]: You missed `t()` here.

Comment (angaither) [inline lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc]: fixed

**Comment by sergiitk** [comment]
:+1:

**Pull request merged**